test/DynamoCoreWpfTests/PresetViewModelTest.cs

```diff
@@ -58,5 +58,37 @@ public void TogglePresetOptionVisibility()
             //Check the Preset option visibility.
             Assert.AreEqual(false, ViewModel.EnablePresetOptions);
         }
+
+        [Test]
+        public void CanCreatePreset()
```

Comment:
Yes. Preset Creation is checked on the View. So, this test checks whether the passed node is an input node. Similiar tests on the model does not have this filter, as it is assumed that only input nodes are passed to create preset.